lib/iris/fileformats/_pyke_rules/fc_rules_cf.krb

```diff
@@ -1029,6 +1029,9 @@ fc_extras
         try:
             iris.unit.as_unit(attr_units)
         except ValueError:
+        # formatting a unicode string causes an exception. Test for that.
+            if isinstance(attr_units, unicode):
+                attr_units = attr_units.encode('ascii', errors='backslashreplace')
```

Comment:
Slight oddity, the same change passes on Master. I haven't looked to see what difference may have caused this incompatibility yet.